sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -863,6 +877,555 @@ SUBROUTINE INITPOST_NETCDF(ncid3d)
           cwm(isa,jsa,l)
       end do 
 
+!=============================
+! For AQF Chemical species
+!=============================
+
+       ! *********** VarName need to be in lower case ************
+       ! === It will cause problem if not use the lower case =====
+       ! *********************************************************
+
+       VarName='o3mr'
```

Comment:
@hsinmulin-NOAA The 'o3mr' is already read in the current setting. Please remove your change.